src/shopilingus/FoodProduct.java

```diff
@@ -0,0 +1,23 @@
+package shopilingus;
+
+import java.util.List;
+
+public class FoodProduct extends Product {
+
+    private List<String> ingredients;
+
+    public FoodProduct(String name, double price, List<String> ingredients){
+        super(name, price, "Food products", 0.07);
+        this.ingredients = ingredients;
+    }
+
+    @Override
+    public String getFullName() {
+        return super.getFullName() + " (" + String.join(", ", ingredients) + " )";
+    }
+
+    @Override
+    public double getPriceForCompany(){
+        return (1-this.getTax())*super.getPriceForCompany(); //???
```

Comment:
* ctrl + alt + L (formatting)
* remove comment (//???)
* for clothing products `getPriceForCompany` doesn't change actually, it's calculated in the same way as for normal products but only with different tax (which we already passed to super constructor in 10th line). so this override method can be removed